algolia/opt/disable_standard_entries.go

```diff
@@ -0,0 +1,63 @@
+// Code generated by go generate. DO NOT EDIT.
+
+package opt
+
+import (
+	"encoding/json"
+	"reflect"
+)
+
+// DisableStandardEntriesOption is a wrapper for an DisableStandardEntries option parameter. It holds
+// the actual value of the option that can be accessed by calling Get.
+type DisableStandardEntriesOption struct {
+	value map[string]map[string]bool
```

Comment:
I noticed there's a type def `DictionaryName` that encodes the enum for this first key of the map. Is it on purpose that this type is not used here?

Reply:
As I noticed above, it's a generated property, so I used the embedded types.

Reply:
**Update**: 
I tried different approaches:
- Move `DictionaryName` to the `opt` package – this introduces a lot of overhead in the generation logic
- Add `DisableStandardEntriesOption` struct manually directly in the `search` package – this introduces a lot of copy-pasted code (raw value wrapper/unwrapper, marshal/unmarshal methods, equals method) that can be generated. And it risks to become even worse if a new per-dictionary parameters will be introduced. 
So, I think that the usage of the raw string as a dictionary name is justified here.

Reply:
SGTM